### DISCUSSION TITLE
Support for api keys

### DISCUSSION THREAD

**Discussion by thunder-007** [microsoft/presidio-research#102] [Ideas]
How add support for api keys like lets collected leaked api keys for famous cloud services like s3,openai keys,ssh pirvate keys.

**Comment by omri374** [comment]
It should be quite straightforward to use something like 'detect-secrets' as part of Presidio
https://github.com/Yelp/detect-secrets/blob/6cdb79bbbd5622d99329a7129e4612f72d6350cd/detect_secrets/main.py#L92

**Reply by thunder-007** [reply]
Thank you for suggesting detect-secrets. it's working good
